_tests/integration/manual_config_test.go

```diff
@@ -127,15 +138,33 @@ var customConfigResultYML = fmt.Sprintf(`options:
     env_key: GRADLE_BUILD_FILE_PATH
     value_map:
       _:
-        title: Gradle task to run
-        env_key: GRADLE_TASK
+        title: Path to the Android project root directory
+        env_key: PROJECT_ROOT_DIR
```

Comment:
this if the gralew fiel's directory, do not need to wire out as a separate option. This should be under gradlew option.